addon/helpers/object.js

```diff
@@ -25,7 +21,7 @@ export default MicroState.extend({
 
   actions: {
     recompute(current, params, options) {
-      return options;
+      return assign({}, options);
```

Comment:
But we shouldn't be adding properties to this object, right? Just using as the prototype for the object that _will_ get the properties defined on it.